runner/repl.js

```diff
@@ -79,6 +79,15 @@ const memoryToString = mem => {
 };
 
 let prev = '';
+
+function setLastEvalExp(prev, ret) {
+  const lastEvalExp = `let _ = ${ret != null ? ret : "undefined"};\n`;
```

Comment:
This will break for anything that's not just a literal.

Reply:
you rigth, thanks🥺